support-frontend/assets/pages/digital-subscription-landing/digitalSubscriptionLandingProps.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import type { ProductPrices } from 'helpers/productPrice/productPrices';
+import { getGlobal, getProductPrices, getPromotionCopy } from 'helpers/globals';
+import type { PromotionCopy } from 'helpers/productPrice/promotions';
+import {
+  detect as detectCountryGroup,
+  type CountryGroupId,
+} from 'helpers/internationalisation/countryGroup';
+import { detect as detectCountry, type IsoCountry } from 'helpers/internationalisation/country';
+import {
+  detect as detectCurrency,
+  type IsoCurrency,
+} from 'helpers/internationalisation/currency';
+import { init as initAbTests, type Participations } from 'helpers/abTests/abtest';
+import { getSettings } from 'helpers/globals';
+
+export type DigitalLandingPropTypes = {|
+  countryId: IsoCountry;
+  countryGroupId: CountryGroupId;
+  currencyId: IsoCurrency;
+  participations: Participations;
+  productPrices: ?ProductPrices;
+  promotionCopy: ?PromotionCopy;
+  orderIsAGift: ?boolean;
+|}
+
+const countryGroupId = detectCountryGroup();
+const countryId = detectCountry();
+
+export const digitalLandingProps: DigitalLandingPropTypes = {
```

Comment:
Just a thought regarding testability, would this file be more testable if it exported a function that returned an object rather than exporting an object? Importing this module as a dependency will trigger the side affect of calling `detectCountryGroup`, `detectCountry`, `detectCurrency` etc, wish I believe would make them harder to mock?